tests/integration/cli/test_accounts.py

```diff
@@ -38,21 +44,39 @@ def temp_account():
     return Account.from_key(bytes.fromhex(PRIVATE_KEY))
 
 
+@pytest.fixture()
+def temp_account_mnemonic():
+    Account.enable_unaudited_hdwallet_features()
+    return Account.from_mnemonic(MNEMONIC)
+
+
 @run_once
 def test_import(ape_cli, runner, temp_account, temp_keyfile_path):
     assert not temp_keyfile_path.is_file()
     # Add account from private keys
-    result = runner.invoke(ape_cli, ["accounts", "import", ALIAS], input=IMPORT_VALID_INPUT)
+    result = runner.invoke(ape_cli, ["accounts", "import", ALIAS], input=IMPORT_VALID_INPUT_PRIVKEY)
```

Comment:
I actually think this would be more clear if it was written down directly instead of using a constant:
```suggestion
    result = runner.invoke(
        ape_cli,
        ["accounts", "import", ALIAS],
        input=f"0x{PRIVATE_KEY}\n{PASSWORD}\n{PASSWORD}\n",
    )
```

Reply:
OK we can do this but I think the tests should all be redone with this thematic element then? I'll redo them all with this theme if that's acceptable to you?

Reply:
Also just to be aware this seems like code reuse but if it seems clearer then I'm more than happy to do it this way.

Reply:
I would say if those constants are only used once or twice is not worth it to make them

Reply:
@unparalleled-js please let me know your thoughts :)

Reply:
I have removed the constants here but I can still remove the join as well if that will make it more readable. 
https://github.com/ApeWorX/ape/pull/1142/commits/9857d0f40e958828a2f0f96ac8b6b92a14bcf54c

Reply:
removed more constants https://github.com/ApeWorX/ape/pull/1142/commits/682939aa60a541a6c8ae20e04da7214963a30dac

Reply:
Clarity is more important than code-reuse in tests!